### ISSUE TITLE
linter analysis

### ISSUE THREAD

**Issue by greenozon** [noct-ml/proxy_dll#3]
some output from C++ linter

High level

![image](https://github.com/Fyyre/proxy_dll/assets/765343/31b476c0-dd82-4466-9449-5be28892474e)

Mid
![image](https://github.com/Fyyre/proxy_dll/assets/765343/9649f205-f468-49ba-97a6-edf7d0f30bc1)

Low
![image](https://github.com/Fyyre/proxy_dll/assets/765343/f7084b43-0d5d-486a-b5ac-9f877745ef47)

**Comment by Fyyre** [comment]
Thanks, I have PVS Studio as well. If it bothers you, feel free to correct it yourself and submit to this repo =)

**Comment by greenozon** [comment]
Yeah, some items sounds dangerous.. will try to address it and get back here soon
